### PR TITLE
Feat/animal assessment documentation

### PR DESCRIPTION
**Pull request by AndyDe-Aldent**
----
[AB#20510](https://dev.azure.com/rezaresystems/15020b92-fee6-4202-956f-9036d1e1da84/_workitems/edit/20510)

Checked spelling using word.
Linked copied from existing instances.

Descriptions have come from the schema descriptions.

----

## PR Checklist
- [ ] Documentation has been checked by at least one team member
- [x] Links work to navigate documentation
- [ ] ~~Images stored in `_images`~~
- [x] Spell checked

> If some of the above are not completed why?
> **REASON HERE**

### CODE REVIEW

**Review by sb226 — COMMENTED** [review]
Just a little change that's all

File: docs/resource-types/livestock/animal-assessment.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: Animal Assessment Event
+menu_order: 1
+post_status: publish
+post_excerpt: Livestock - Animal Assessment Events
```
[the remark this answers is not in the document]
Comment (AndyDe-Aldent) [reply docs/resource-types/livestock/animal-assessment.md]: done

File: docs/resource-types/livestock/animal-assessment.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+taxonomy:
+  category:
+    - api
+    - data
+  post_tag:
+    - api
+---
+
+# Resource Type: Livestock - Animal Assessment Events
```
[the remark this answers is not in the document]
Comment (AndyDe-Aldent) [reply docs/resource-types/livestock/animal-assessment.md]: done

File: docs/resource-types/livestock/animal-assessment.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  "@self": "string",
+  "location": {...},
+  "meta": {...},
+  "resourceType": "/livestock/animal-assessment-event",
+  "eventDateTime": "2023-01-03T11:00:00Z",
+  "id": "string",
+  "animal": {...},
+  "metric": {...},
+  "assessment": "string",
+  "score": "number"
```
Comment (sb226) [inline docs/resource-types/livestock/animal-assessment.md:64]: Missing: `Responsible`, `Remark`, `TraitLabel`, `ComtemporaryGroup`

**Review by sb226 — COMMENTED** [review]
Sorry last one

**Review by sb226 — APPROVED** [review]
⚾ 🏠 🏃

**Pull request merged**